book/reference/glossary.md

```diff
@@ -0,0 +1,72 @@
+  environments. Binder is a 100% open source infrastructure that is run by
+  members of the Jupyter community. The underlying technology behind the Binder
+  project is BinderHub.
+
+[BinderHub](https://binderhub.readthedocs.io)
+  The underlying technology of mybinder.org, BinderHub is an open source tool
+  that utilizes a JupyterHub`= in order to provide live, reproducible
+  interactive computing environments that users define on GitHub.
+
+[Conda](https://docs.conda.io)
+  Package, dependency and environment management for any language—Python, R,
+  Ruby, Lua, Scala, Java, JavaScript, C/ C++, FORTRAN, and more.
+
+[Docker](https://www.docker.com)
+  Docker provides the ability to package and run an application in a loosely
+  isolated environment called a container. It is widely used for creating
+  reproducible software environments to run code on different computers.
+
+[Git](https://git-scm.com)
+  A popular version control system that is used in many open source software
+  projects to manage their software code base.
+
+[GitHub](https://github.com)
+  Provider of Internet hosting for software development and distributed version
+  control using the "git" command line tool.
+
+[Project Jupyter](https://jupyter.org)
+  Project Jupyter (name derived from "JUlia PYThon and R") exists to develop
+  open-source software, open-standards, and services for interactive computing
+  across dozens of programming languages.
+
+[Jupyter Book](https://jupyterbook.org/intro.html)
+  Jupyter Book is an open source project for building beautiful,
+  publication-quality books and documents from computational material.
+
+[JupyterHub](https://jupyterhub.readthedocs.io)
+  A core open source tool from the Jupyter community, JupyterHub allows you to
+  deploy an application that provides remote data science environments to
+  multiple users. It can be deployed in the cloud, or on your own hardware.
+
+[JupyterLab](https://jupyterlab.readthedocs.io)
+  JupyterLab is the next-generation web-based user interface for Project Jupyter
+  intended to replace the JupyterNotebook interface.
+
+[Jupyter Notebook](https://jupyterbook.org)
+  open-source web application that allows you to create and share documents that
+  contain live code, equations, visualizations and narrative text.
+
+[MyST](https://myst-parser.readthedocs.io)
+  Markedly Structured Text (MyST) is a rich and extensible flavor of Markdown
+  meant for technical documentation and publishing. It is used by Jupyter Book.
+
+```
+
+## Topical (Use Case Specific)
+```{glossary}
+[basal slip](https://jupyterlab.readthedocs.io)
+  JupyterLab is the next-generation web-based user interface for Project Jupyter
+  intended to replace the JupyterNotebook interface.
+
+[subglacial water](https://jupyterlab.readthedocs.io)
+  JupyterLab is the next-generation web-based user interface for Project Jupyter
+  intended to replace the JupyterNotebook interface.
```

Comment:
Need to find the correct definition. I'd use the classic Cuffey and Patterson "Physics of Glaciers", but that seems to be print only? There's another good glossary by Cogley et al 2011 at https://glaciers.gi.alaska.edu/sites/default/files/Cogley2011_Glossary.pdf, but it's more mass balance specific.

Reply:
Agreed. Template import and adds look great so far! I've only seen Cuffey and Patterson "Physics of Glaciers" in print, but that doesn't mean we can't cite it!

Reply:
I addressed this comment with [this PR](https://github.com/geo-smart/deepicedrainpipe/pull/18).